### ISSUE TITLE
Porting to Swift

### ISSUE THREAD

**Issue by martyu** [itsmeichigo/ICGVideoTrimmer#39]
I don't think github has a "discussion" secion, so I'm putting this here...

The Objective-C-ness of this is driving me crazy.  And to a lesser extent, the non-autolayout-ness.  I'm rewriting to Swift in a...new project I guess?  I guess it doesn't make sense to fork it since it will be all new code?  Has anyone already done this, or is there any reason I shouldn't?

**Comment by itsmeichigo** [comment]
Hi, I don't think anyone did that - so please feel free to port it to swift. I can help with that too if you need another hand!

**Comment by martyu** [comment]
Sweet I’ll send you the PR. Also going to see if I can work in autolayout. Or at the very least refactor out that ‘resetSubviews’ method. Seems like most of it could be moved to ‘init’ and ‘layoutSubviews’

**Comment by itsmeichigo** [comment]
Yes I agree that autolayout would be great and should be used instead of my current messy screen size calculation. But are you putting the swift code in this same repo? I think a separate repo would be neater.

**Comment by martyu** [comment]
Yeah that's why I was saying it probably wouldn't be good to fork it even.  I guess it would be a new repo and I would just link back to this repo in the Readme for reference?

**Comment by itsmeichigo** [comment]
Yes that works - let me know if you need any help.
